pandas/core/generic.py

```diff
@@ -6542,23 +6542,18 @@ def copy(self, deep: bool = True) -> Self:
 
         When ``deep=False``, a new object will be created without copying
         the calling object's data or index (only references to the data
-        and index are copied). Any changes to the data of the original
-        will be reflected in the shallow copy (and vice versa).
+        and index are copied). With Copy-on-Write enabled by default,
```

Comment:
Since CoW can't be disabled, I think we should remove the "enabled by default" bit.
```suggestion
        and index are copied). With Copy-on-Write,
```

Reply:
I've removed the phase `enabled by default`